packages/docusaurus/src/client/App.tsx

```diff
@@ -17,6 +17,35 @@ import PendingNavigation from './PendingNavigation';
 
 import './client-lifecycles-dispatcher';
 
+function isAnyCSSLoaded() {
+  const stylesheets = document.styleSheets;
+  for (let i = 0; i < stylesheets.length; i += 1) {
+    try {
+      if (
+        stylesheets[i].cssRules != null &&
+        stylesheets[i].cssRules.length >= 0
+      ) {
+        return true;
+      }
+    } catch (e) {}
+  }
+  return false;
+}
```

Comment:
you shouldn't do that,  because it won't trigger any react re-render on css load.

Instead do something like I suggested:

```js
import "./hideCSSLoadingWarning.css"
```

```css
.css-loading-warning {
  display: none;
}
```

If the CSS of the site loads, the message will automatically hide, so it will remain visible only if css does not load.

Reply:
Oh now I get it!
Sorry I think I had misunderstood it the first time.
I'll try this, thank you!